.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - "iojs"
+  - "4"
   - "0.12"
-  - "0.11"
```

Comment:
Cool, this is more so I know for the other modules. I want to standardise on this.